include/aikido/control/ros/Conversions.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef AIKIDO_CONTROL_ROS_CONVERSIONS_HPP_
+#define AIKIDO_CONTROL_ROS_CONVERSIONS_HPP_
+#include <memory>
+#include <trajectory_msgs/JointTrajectory.h>
+#include <aikido/statespace/dart/MetaSkeletonStateSpace.hpp>
+#include <aikido/trajectory/Spline.hpp>
+#include <map>
+
+namespace aikido {
+namespace control {
+namespace ros {
+
+/// Converts a ROS JointTrajectory into an aikido's Spline trajectory.
+/// This method only handles single-DOF joints.
+/// \param[in] _space MetaSkeletonStateSpace for Spline trajectory.
+//              Subspaces must be either 1D RnJoint or SO2Joint.
+/// \param[in] _jointTrajectory ROS JointTrajectory to be converted.
+/// \return Spline trajectory.
+std::unique_ptr<aikido::trajectory::Spline> convertJointTrajectory(
+  const std::shared_ptr<
+    aikido::statespace::dart::MetaSkeletonStateSpace>& space,
+  const trajectory_msgs::JointTrajectory& jointTrajectory);
+
+/// Converts Aikido Trajectory to ROS JointTrajectory.
+/// Supports only 1D RnJoints and SO2Joints.
+/// \_param[in] trajectory Aikido trajectory to be converted.
+/// \_param[in] indexMap Mapping between trajectory's joints and ros joints.
+/// \_param[in] timestep Timestep between two consecutive waypoints.
+const trajectory_msgs::JointTrajectory convertTrajectoryToRosTrajectory(
```

Comment:
I would like to suggest patterning the two convert function names. `convertJointTrajectory()` doesn't convey the original and target type in the function name unlike to the latter function name, and `convertTrajectoryToRosTrajectory()` uses only `Trajectory` to refer `JointTrajectory` unlike to the former function name.

We could put both of original and target type name to the function name like `convertJointTrajectoryToRosJointTrajectory()` but this might be too long. So my preference is including only the goal type to the function name so that we can call the correct conversion functions depending on the target type. We don't need to worry about the original type since functions can overload for the argument types, which is the original type, but not for the return type.

**TL;DR**: I suggest renaming the two function names to something like:
```cpp
convertToSplineJointTrajectory()
convertToRosJointTrajectory()

// or
toSplineJointTrajectory()
toRosJointTrajectory()
```